docs/topics/wasm/wasm-configuration.md

```diff
@@ -170,6 +179,14 @@ kotlin {
 
 Keep in mind that enabling this option increases the application size.
 
+### Fully qualified names
+
+On Kotlin/Wasm targets, fully qualified names (FQNs) are available at runtime without any additional configuration.
+This means that the `KClass.qualifiedName` property is enabled by default.
+
+Having FQNs improves code portability from JVM to Wasm targets and makes runtime errors more informative by displaying 
```

Comment:
I'd like to use more common verbs here:

**Using** FQNs improves code portability from JVM to Wasm targets and makes runtime errors more informative by **showing** the full qualified name.

WDYT?